helm/basemap/templates/cronjobs.yaml

```diff
@@ -98,11 +99,18 @@ spec:
               requests:
                 memory: {{ .Values.resources.db.requests.memory | quote }}
                 cpu: {{ .Values.resources.db.requests.cpu | quote }}
+            # Comment by: Jamal Shahverdiyev
+            # I kept these lines just to be sure, if the changes due PVC folder cleaning will not work I will try to uncomment these lines and test with them
+            # lifecycle:
+            #   preStop:
+            #     exec:
+            #       command: ["sh", "-c", "rm -rf /var/lib/postgresql/data/*"]
           volumes:
           - name: {{ .Values.envName }}-basemap-processor-pvc
             persistentVolumeClaim:
               claimName: {{ .Values.envName }}-basemap-processor-pvc
           - name: {{ .Values.envName }}-basemap-postgres-pvc
+            emptyDir: {}
```

Comment:
Is it allowed to combine PVC provisioning with emptyDir provisioning? https://www.alibabacloud.com/blog/kubernetes-volume-basics-emptydir-and-persistentvolume_594834 suggests that you can only choose one of these.

Will emptyDir provision provide enough disk space if not configured? Will the space be on the nvme drive?

Reply:
I read this document. In the second example "2) Pod with 3 Containers Sharing emptyDir Use" they are using shared PVC between containers but in our case we use 2 different for 2 different containers. So space between them not issue. But about emptyDir with PVC provisioning I am not sure too. Give me time pls I will simulate it just to be sure. After that I will write answer here.